mmaction/datasets/pipelines/augmentations.py

```diff
@@ -1202,8 +1202,7 @@ def __call__(self, results):
         modality = results['modality']
 
         if modality == 'RGB':
-            n = len(results['imgs'])
-            h, w, c = results['imgs'][0].shape
+            n, h, w, c = np.shape(results['imgs'])
```

Comment:
np.shape implicitly calls `asarray`. I'm not sure if this induces any data copy.

Could you pls comapre the speed of `np.shape(results['imgs'])` and the original version?

Reply:
I've rolled back， and `np.shape` is very slow indeed.